doc/devkit.dox.in

```diff
@@ -1,16 +1,16 @@
 # Doxygen configuration file for generating
-# the standard developer RINGMesh and geogram documentation
+# the standard developer RINGMesh documentation excluding geogram
```

Comment:
Is it necessary that geogram is excluded. It is weird to highlight it. Or you should also say that zlib, minizip and tinyxml2 are excluded. I would remove the "excluding geogram" and just say it is the doc of RINGMesh sources.